spectator-agent/src/main/resources/bitbucket.conf

```diff
@@ -0,0 +1,132 @@
+// http://confluence.atlassian.com/bitbucketserver/enabling-jmx-counters-for-performance-monitoring-776640189.html
+netflix.spectator.agent.jmx {
+  mappings = ${?netflix.spectator.agent.jmx.mappings} [
+    {
+      query = "com.atlassian.bitbucket.thread-pools:name=*"
+      measurements = [
+        {
+          name = "bitbucket.{name}.MaximumPoolSize"
+          value = "{MaximumPoolSize}"
+          tags = [ ]
+        },
+        {
+          name = "bitbucket.{name}.PoolSize"
+          value = "{PoolSize}"
+          tags = [ ]
+        },
+        {
+          name = "bitbucket.{name}.ActiveCount"
+          value = "{ActiveCount}"
+          tags = [ ]
+        },
+        {
+          name = "bitbucket.{name}.QueueLength"
+          value = "{QueueLength}"
+          tags = [ ]
+        }
+      ]
+    },
+    {
+      query = "com.atlassian.bitbucket:name=ScmStatistics"
+      measurements = [
+        {
+          name = "bitbucket.scm.Pulls"
```

Comment:
Per the docs Pulls/Pushes are monotonic counter since the process was started. They should use `atlas.dstype=rate` with specify the counter config option, e.g.:

https://github.com/Netflix/spectator/blob/master/spectator-agent/src/main/resources/cassandra.conf#L1579